src/styles/Stylepack/_reset.scss

```
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
```

```suggestion
* {
  box-sizing: border-box;
```
Do you really want to apply these styles to ALL elements?
It's not good for performance, better specify the exact tags and leave only box-sizing here